packages/communication-react/review/stable/communication-react.api.md

```diff
@@ -1964,7 +1964,7 @@ export const ParticipantItem: (props: ParticipantItemProps) => JSX.Element;
 
 // @public
 export interface ParticipantItemProps {
-    displayName: string;
+    displayName?: string;
```

Comment:
Did we add this field in the current stable release?

@PorterNan to double check this won't cause any problems with `usePropsFor` typing detection logic.